storage/container_storage_interface/persistent-storage-csi-azure.adoc

```diff
@@ -31,6 +31,12 @@ After full migration, in-tree plugins will eventually be removed in later versio
 
 include::modules/persistent-storage-csi-azure-disk-sc-zrs.adoc[leveloffset=+1]
 
+ifndef::openshift-rosa,openshift-dedicated[]
+include::modules/persistent-storage-byok.adoc[leveloffset=+1]
+
+For information about installing with user-managed encryption for Azure, see xref:../../installing/installing_azure/enabling-user-managed-encryption-azure.adoc[Enabling user-managed encryption for Azure] and xref:../../installing/installing_azure/installing-azure-network-customizations.adoc#finalizing-encryption_installing-azure-network-customizations[Finalizing user-managed encryption after installation].
```

Comment:
Based on this sentence: "Finalizing user-managed encryption after installation", it seems that Azure needs some extra steps to do after installation, and that `installing-azure-network-customizations.adoc` must document them. I failed to find any relevant info in the latter. Is it really true that Azure (unlike AWS and GCP) needs some networking customization for finalizing user-managed encryption?

Reply:
You are correct. I was questioning that with the install writer and he agreed, but I didn't get around to removing that second link.